.github/workflows/cd-pipeline.yml

```diff
@@ -124,3 +124,75 @@ jobs:
           VIB_ENV_REGISTRY_URL: ${{ secrets.PROD_MARKETPLACE_REGISTRY_URL }}
           VIB_ENV_REGISTRY_USERNAME: ${{ steps.get-registry-credentials.outputs.marketplace_user }}
           VIB_ENV_REGISTRY_PASSWORD: ${{ steps.get-registry-credentials.outputs.marketplace_passwd }}
+      - name: Install s3cmd
+          run: sudo apt-get install -y s3cmd
+      - name: 'Getting 3rd party packages for OSSPI for ${{ steps.get-container-metadata.outputs.name }}: ${{ steps.get-container-metadata.outputs.tag }}'
+        if: ${{ steps.get-container-metadata.outputs.result == 'ok' }}
+        env:
+          # Path with docker resources
+          VIB_ENV_PATH: ${{ matrix.container }}
+        run: |
+          # If it's set from outside, can be changed
+          ARCH="${ARCH:-amd64}"
+          IMAGE_NAME=$(grep -oE "org.opencontainers.image.ref.name=\".+\"" ${{ matrix.container }}/Dockerfile | sed -nr "s|org.opencontainers.image.ref.name=\"(.+)\"|\1|p")
```

Comment:
Why don't use `${{ steps.get-container-metadata.outputs.tag }}`?

Reply:
I did it thinking on being an _isolated_ step